.github/workflows/deploy_staging_TestNet_UI.yml

```diff
@@ -0,0 +1,57 @@
+name: 2wp-app deploy UI for staging TestNet
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  checkout-and-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get tag name.
+        id: get_tag_name
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ steps.get_tag_name.outputs.VERSION }}
+
+      - name: Setup NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build site
+        run: >
+          VUE_APP_COIN='test'
+          VUE_APP_MANIFEST_EMAIL='user@rsk.co'
+          VUE_APP_MANIFEST_APP_URL='https://2wp-app.testnet.rsk.co'
+          VUE_APP_API_BASE_URL='https://2wp-api.testnet.rsk.co'
```

Comment:
Same observation here, point to staging-testnet `API` url

Reply:
You are right.
Thanks.
Fixed.